Remotion/ObjectBinding/Web/UI/Controls/BocTextValueImplementation/BocTextValueAutoCompleteAttribute.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Remotion.ObjectBinding.Web.UI.Controls.BocTextValueImplementation
+{
+  /// <summary>
+  /// Defines possible standard autocomplete attributes to be used with <see cref="TextBoxStyle.AutoComplete"/>.
+  /// For a clear documentation about all the attributes, please visit the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete">MDN docs</a> for the HTML attribute.
+  /// </summary>
+  public class BocTextValueAutoCompleteAttribute
```

Comment:
This gets moved to UI/Controls. In Implementation we only have stuff you don't need when using the controls (i.e. stuff for infrastructure developers)